### PR TITLE
feat(chat): redesign header tooltips (Issue #79)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

redesign header tooltips

Issues:

- Issue #79

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2790.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2790.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2790.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2790.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/ChatHeader/HeaderSettingsTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      (lineClamp) => window.innerHeight <= lineClamp.threshold,
+    )?.class || DEFAULT_SM_LINE_CLAMP;
+
+  return (
+    <div
+      className="grid max-w-[880px] grid-cols-1 p-2"
+      data-qa="chat-settings-tooltip"
+    >
+      <div className="font-semibold">{t('Change conversation settings')}:</div>
+      <div className="mt-3 grid max-w-full grid-cols-[auto,1fr] gap-x-4 gap-y-2">
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ChatHeader/HeaderSettingsTooltip.tsx:46]: why not `grid-cols-1` instead of `grid-cols-[auto,1fr]`?

**Review by IlyaBondar — APPROVED** [review]

**Review by irinakartun — APPROVED** [review]

**Pull request merged**